Serverless/Azure Functions.md

```diff
@@ -1,2 +1,151 @@
-## Azure Functions
+Azure Functions is a Function-as-a-Service (“FaaS”), i.e. a platform for running configurable blocks of code that run in response to some event.
+Functions support several different programming languages such as C# and JavaScript, F#, Java, Python, and PowerShell.
+
+In general, Azure Functions are a serverless solution that provide:
+* Writing less code - Allows developer to focus on the business logic implementation and utilize the ready-to-use infrastructure features. 
+* Maintain less infrastructure - Cloud infrastructure providers manage servers, security and scaling. Functions are scaled automatically.
+* Save on costs - No need to buy resources in advance - pay as you go.
```

Comment:
Maybe reword the first part to something like "Main benefits of using Azure Functions and serverless solutions are :" because the two bullets don't match wording of "that provide : ".